### PR TITLE
create a boilerplate/example .py test

### PR DESCRIPTION
**Pull request by gandrewstone**
create a boilerplate .py test that people can use to create their own tests -- this is NOT actually a test so is not added to pull_tester

### CODE REVIEW

File: qa/rpc-tests/test_template.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+# This is a template to make creating new QA tests easy.
+# You can also use this template to quickly start and connect a few regtest nodes.
+
+import time
+import sys
+if sys.version_info[0] < 3:
+    raise "Use Python 3"
+import logging
+logging.basicConfig(format='%(asctime)s.%(levelname)s: %(message)s', level=logging.INFO,stream=sys.stdout)
```
Comment (sickpig) [inline qa/rpc-tests/test_template.py:14]: nit: mix used of spaces after comma in parameter of this function

**Comment by sickpig** [comment]
a part from the nit I find this boilerplate very useful.

**Review by sickpig — APPROVED** [review]

**Comment by awemany** [comment]
Just wondering: Should this maybe also include a generic node setup?

**Comment by sickpig** [comment]
@gandrewstone this ready to get merged

**Comment by gandrewstone** [comment]
@awemany, you mean a python node?

**Comment by awemany** [comment]
Yeah.

**Pull request merged**